### ISSUE TITLE
1.20.1 Field Manual says "invalid"

### ISSUE THREAD

**Issue by DefinedAscent** [oitsjustjose/Geolosys#325]
## Versions:

- Minecraft Forge: 47.3.0
- Geolosys: 1.20.1 - 7.0.10

## What happens: 
Field Manual doesn't show up in JEI but is craftable. However the book says it is invalid from Patchouli

## What should happen: 
Field Manual should work like other Patchouli books

## Additional Comments: 
Other Patchouli books do work, like Dynamic Trees for example, and I have tested this. I assume a mod would conflict and be the reason, but considering everything else for the mod works but not the manual it is a little strange.

![2024-12-28_15 45 35](https://github.com/user-attachments/assets/a61d5fd7-1393-4004-95dd-ba1b889df9fb)

**Comment by DefinedAscent** [comment]
I went into a new 1.20.1 world and did some testing with only Geolysis and get this error in my log:

```
[17:20:32] [Worker-Main-1/ERROR] [patchouli/]: Failed to load book geolosys:field_manual defined by mod geolosys, skipping
java.lang.IllegalArgumentException: Book geolosys:field_manual has use_resource_pack set to false. This behaviour was removed in 1.20. The book author should enable this flag and move all book contents clientside to /assets/, leaving the book.json in /data/. See https://vazkiimods.github.io/Patchouli/docs/upgrading/upgrade-guide-120 for details.
	at vazkii.patchouli.common.book.Book.<init>(Book.java:145) ~[Patchouli-1.20.1-84-FORGE.jar%23131!/:1.20.1-84-FORGE] {re:mixin,re:classloading}
	at vazkii.patchouli.common.book.BookRegistry.loadBook(BookRegistry.java:90) ~[Patchouli-1.20.1-84-FORGE.jar%23131!/:1.20.1-84-FORGE] {re:mixin,re:classloading}
	at vazkii.patchouli.common.book.BookRegistry.lambda$init$3(BookRegistry.java:75) ~[Patchouli-1.20.1-84-FORGE.jar%23131!/:1.20.1-84-FORGE] {re:mixin,re:classloading}
	at java.util.HashMap.forEach(HashMap.java:1421) ~[?:?] {}
	at vazkii.patchouli.common.book.BookRegistry.init(BookRegistry.java:70) ~[Patchouli-1.20.1-84-FORGE.jar%23131!/:1.20.1-84-FORGE] {re:mixin,re:classloading}
	at vazkii.patchouli.forge.common.ForgeModInitializer.onInitialize(ForgeModInitializer.java:76) ~[Patchouli-1.20.1-84-FORGE.jar%23131!/:1.20.1-84-FORGE] {re:classloading}
	at vazkii.patchouli.forge.common.__ForgeModInitializer_onInitialize_FMLCommonSetupEvent.invoke(.dynamic) ~[Patchouli-1.20.1-84-FORGE.jar%23131!/:1.20.1-84-FORGE] {re:classloading,pl:eventbus:B}
	at net.minecraftforge.eventbus.ASMEventHandler.invoke(ASMEventHandler.java:73) ~[eventbus-6.0.5.jar%23109!/:?] {}
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:315) ~[eventbus-6.0.5.jar%23109!/:?] {}
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:296) ~[eventbus-6.0.5.jar%23109!/:?] {}
	at net.minecraftforge.fml.javafmlmod.FMLModContainer.acceptEvent(FMLModContainer.java:114) ~[javafmllanguage-1.20.1-47.3.0.jar%23134!/:?] {}
	at net.minecraftforge.fml.ModContainer.lambda$buildTransitionHandler$4(ModContainer.java:125) ~[fmlcore-1.20.1-47.3.0.jar%23133!/:?] {}
	at java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1804) ~[?:?] {}
	at java.util.concurrent.CompletableFuture$AsyncRun.exec(CompletableFuture.java:1796) ~[?:?] {}
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373) ~[?:?] {}
	at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182) ~[?:?] {}
	at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655) ~[?:?] {}
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622) ~[?:?] {}
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165) ~[?:?] {}
```

I was able to edit the resource path structure and get the book to at least load, but the current entry pointers no longer work. And I was having small errors with each entry.
![image](https://github.com/user-attachments/assets/4c8aa29b-2e73-4ec3-ad8a-e8d7476eba05)

**Comment by oitsjustjose** [comment]
Fixed in 19f3f3954cc51038393a04f465d98ccf03a3d57e :)
